pywinauto/windows/uia_element_info.py

```diff
@@ -55,6 +56,28 @@ def elements_from_uia_array(ptrs, cache_enable=False):
     return elements
 
 
+def is_element_satisfies_criteria(element, process=None, class_name=None, name=None, control_type=None,
+                                  content_only=None, **kwargs):
+    """Check if element satisfy filter criteria"""
```

Comment:
Also should be:
"""Check if element satisfies filter criteria"""

Reply:
done